public/js/reducers/pause.js

```diff
@@ -137,7 +137,9 @@ function getShouldIgnoreCaughtExceptions(state) {
 }
 
 function getFrames(state) {
-  return state.pause.get("frames") || [];
+  // frames should always be an Immutable Seq
+  // Return an empty Seq if the state is null
+  return state.pause.get("frames") || fromJS([]);
```

Comment:
Yeah, the default is `null`, so maybe it'd be better to just have the selector return the exact state and have the React view do a Boolean check for whether `frames` isn't `null`.